src/app/login-modal/login-modal.html

```diff
@@ -1,41 +1,80 @@
 <biggive-popup-standalone>
-  <h2 mat-dialog-title>Log in</h2>
+  <re-captcha
+    #captcha
+    size="invisible"
+    (resolved)="captchaReturn($event)"
+    siteKey="{{ recaptchaIdSiteKey }}"
+  ></re-captcha>
 
-  <mat-dialog-content class="mat-typography">
-    <form [formGroup]="form">
-      <re-captcha
-        #captcha
-        size="invisible"
-        (resolved)="captchaReturn($event)"
-        siteKey="{{ recaptchaIdSiteKey }}"
-      ></re-captcha>
+  <div *ngIf="forgotPassword else logIn">
```

Comment:
I don't know angular well enough to be confident in this, but I wonder if this conditional logic is good here, or if it might be less confusing if we could separate out the forgotPassword and login forms into two separate templates.

Reply:
@bdsl I think this is fine as is. It's just a slight variation in the UI, and the two views are directly related to each other.